src/main/java/frc/robot/Constants.java

```diff
@@ -217,13 +217,15 @@ public static TalonFXConfiguration getConfiguration() {
 
         public static final double CAMERA_HEIGHT = 0.73; // [m]
         public static final double TARGET_HEIGHT_FROM_GROUND = 2.65; // [m] Pefzener 2.62
+        public static final double BIT_CAMERA_HEIGHT = 0.73; // [m]
+        public static final double BIT_TARGET_HEIGHT_FROM_GROUND = 2.65; // [m] Pefzener 2.62
```

Comment:
The height of the camera should be different as the robot is standing on the עגלת רובוט